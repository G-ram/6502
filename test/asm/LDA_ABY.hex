a9 40 8d 00 01 a0 04 a9 00 b9 fc 00 09 01 8d 00
01 
