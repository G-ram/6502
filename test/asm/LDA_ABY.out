A: 41
X: 00
Y: 04
P: 00010000
S: 00
PC: 611
100 : 41
101 : 00
